tensorflow_addons/optimizers/rectified_adam.py

```diff
@@ -131,7 +131,7 @@ def __init__(
         self._set_hyper("decay", self._initial_decay)
         self._set_hyper("weight_decay", weight_decay)
         self._set_hyper("sma_threshold", sma_threshold)
-        self._set_hyper("total_steps", int(total_steps))
+        self._set_hyper("total_steps", float(total_steps))
```

Comment:
If the weights are added in floating point, do we really need `float(xxx)` here?